src/GovUk.Education.ExploreEducationStatistics.Content.Api.Tests/Controllers/DataSetFilesControllerTests.cs

```diff
@@ -95,6 +95,51 @@ await TestApp.AddTestData<ContentDbContext>(context =>
                 AssertResultsForExpectedReleaseFiles(publication1Release1Version1Files, pagedResult.Results);
             }
 
+            [Fact]
+            public async Task FilterByGeographicLevel_Success()
+            {
+                var (publication1, publication2) = _fixture
+                    .DefaultPublication()
+                    // Publications each have a published release version
+                    .WithReleases(_fixture.DefaultRelease(publishedVersions: 1)
+                        .Generate(1))
+                    .WithTheme(_fixture.DefaultTheme())
+                    .GenerateTuple2();
+
+                var publication1Release1Version1Files = _fixture.DefaultReleaseFile()
+                    .WithReleaseVersion(publication1.ReleaseVersions[0])
+                    .WithFiles(_fixture.DefaultFile(FileType.Data)
+                    .WithDataSetFileMeta(_fixture.DefaultDataSetFileMeta()
+                        .WithGeographicLevels(
+                            [GeographicLevel.Country, GeographicLevel.LocalAuthority, GeographicLevel.Institution]))
+                    .WithDataSetFileGeographicLevels(
+                        [GeographicLevel.Country, GeographicLevel.LocalAuthority, GeographicLevel.Institution])
+                    .GenerateList(1))
+                    .GenerateList();
```

Comment:
This doesn't look quite right and from the test result looks like there's only one file being generated here anyway.

Rather than `var publication1Release1Version1Files` could use `ReleaseFile publication1Release1Version1File` and remove both of these lines, or give more files to publication 1 release version 1 with some different levels.